### PR TITLE
Add expired to getstakeinfo command

### PR DESCRIPTION
**Pull request by alexlyp**

### CODE REVIEW

File: wallet/wallet.go [hunk 1]
```diff
@@ -2825,6 +2825,7 @@ type StakeInfoData struct {
 	Voted         uint32
 	Missed        uint32
 	Revoked       uint32
+	Expired       uint32
```
Comment (chappjc) [inline wallet/wallet.go]: Maybe need something around here too:
https://github.com/decred/dcrwallet/pull/360/files#diff-c35d9ca3f3d9de083c045b115be2f0eeR2857

File: wallet/wallet.go [hunk 2]
```diff
@@ -2975,6 +2976,32 @@ func (w *Wallet) StakeInfo() (*StakeInfoData, error) {
...
 		}
 
+		expiredTicketNum := 0
+		revokeTicketPtrs := make([]*chainhash.Hash, len(revokedTickets))
+		for i := range revokedTickets {
+			revokeTicketPtrs[i] = &revokedTickets[i]
+		}
+
+		// Check the expired ticket pool for the presense of tickets.
+		expiredBitSetBStr, err := chainClient.ExistsExpiredTickets(revokeTicketPtrs)
```
Comment (chappjc) [inline wallet/wallet.go]: Then does [`chainClient.MissedTickets()`](https://github.com/alexlyp/btcwallet/blob/c60d4e2133d4f02d581d9903219cdd6356fa928c/wallet/wallet.go#L3031) no longer include expired? Or does that count need to be decremented?

Comment (alexlyp) [inline wallet/wallet.go]: I guess that's a good point, I was using a inclusive/exclusive view of them.  All expired are missed, but not all missed are expired.

File: wallet/wallet.go (hunk 1 above)
Comment (alexlyp) [inline wallet/wallet.go]: added

**Comment by chappjc** [comment]
tACK
Calling getstakeinfo with dcrctl seemed to trigger a CPU-intensive operation with dcrd, dcrwallet's RPC session timed out, dcrctl said client shutdown, and dcrwallet reconnected shortly after.  Further calls to getstakeinfo worked as expected.  However, this only seemed to happen on one machine.  It worked without hiccup on another.

File: wallet/wallet.go (hunk 2 above)
Comment (alexlyp) [inline wallet/wallet.go]: https://github.com/decred/dcrwallet/issues/371

**Comment by chappjc** [comment]
Is this going in?  I'd like it.

**Pull request merged**